src/Camera.py

```diff
@@ -1,15 +1,55 @@
 #!/usr/bin/python3
-
-import PySpin
-
+import logging
+logger = logging.getLogger(__name__)
+
+try:
+    import PySpin
+except ImportError:
```

Comment:
PySpin is a dependency of this application, so we shouldn't need to handle this import error. I think what you'd instead (*if* we're going for automatic camera assignment, see below) is to handle the case of no cameras connected.